index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">
+    <div class="wrapper">
       <div class="nav-left">
-        <a href="#" class="logo">BankPL</a>
-        <p class="offer-name">Konto 360°</p>
+        <a href="#" class="site-logo">BankPL</a>
+        <span class="offer-name">Konto 360°</p>
       </div>
-      <a href="#" class="nav-icon">&#9776;</a>
+      
+      <button href="#" class="nav-icon">&#9776;</button>

```

Comment:
zmienione a na button, bo to nie jest link - to jest jakis przycisk ktory wykonuje akcje, nie przenosi cie do nowej tresci - oczywiscie odpowiedno ostylowac